android/build.gradle

```diff
@@ -9,31 +9,30 @@ buildscript {
         }
 
         dependencies {
```

Comment:
This changes belongs to [this PR ](https://github.com/onewelcome/sdk-react-native/pull/89/files). It's a small PR so it's not a big issue, though you could point this PR to rnp-27 branch instead of develop in order to hide non-relevant files for this PR.